frontend/pages/topics/index.jsx

```diff
@@ -0,0 +1,49 @@
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import axios from "axios";
```

Comment:
@iacobfred do we need to use [axiosWithoutAuth](https://github.com/ModularHistory/modularhistory/blob/main/frontend/axiosWithoutAuth.ts) here?

Reply:
Hmm.  Looks like the entities page uses vanilla axios as well, and it works fine....